yeelight/main.py

```diff
@@ -1,12 +1,15 @@
 import colorsys
 import json
 import socket
+import logging
+from enum import Enum
 
 from .decorator import decorator
 from .flow import Flow
 
 MUSIC_PORT = 37657
 
+_LOGGER = logging.getLogger(__name__)
```

Comment:
It doesn't log per default, for testing logging you can add the following in the beginning of yeecli:
```python
import logging
logging.basicConfig(level=logging.DEBUG)
```
and you should see the messaging then.